sprint 7/tema 3.js

```diff
@@ -0,0 +1,44 @@
+function dayOfTheWeek(weekDay) {
+  
+  let dayName = parseInt(weekDay);
+
+  switch(dayName) {
+    case 1:
+      return "Monday";
+
+    case 2:
+      return "Tuesday";
+
+    case 3:
+      return "Wednesday";
+
+    case 4:
+      return "Thursday";
+
+    case 5:
+      return "Friday";
+
+    case 6:
+      return "Saturday";
+
+    case 7:
+    case 0:
+      return "Sunday";
+
+    default:
+      return "You have to write a number between 0 and 7.";
+  }
+
+}  
+
+function dayReturn(dayNumber) {
+
+  if (dayNumber === 0 || dayNumber <= 7) {
```

Comment:
In primul rand cred ca ai vrut sa spui
```
if (dayNumber >=0 && dayNumber <=7)
```
In al doilea rand: poti avea si situatii cand dayNumber este 3.4, atunci cum faci? :) 

In general, eroarea se arunca sus, in functia dayOfTheWeek. 
Iar in aceasta functie ar fi suficient sa lasi doar o singura linie:

{
    return "The week day number "+ dayNumber +" is called " + dayOfTheWeek(dayNumber) + ".";
}

Reply:
Am modificat, este in regula sa apara "undefined" cand fac console.log( ) de un numar care nu este intre  0-7?